src/rpcrawtransaction.cpp

```diff
@@ -586,12 +591,21 @@ Value listunspent(const Array& params, bool fHelp)
         if (ExtractDestination(out.tx->vout[out.i].scriptPubKey, address))
         {
             entry.push_back(Pair("address", CBitcoinAddress(address).ToString()));
-            if (pwalletMain->mapAddressBook.count(address))
-                entry.push_back(Pair("account", pwalletMain->mapAddressBook[address]));
+
+            auto item = pwalletMain->mapAddressBook.find(address);
+
+            if (item != pwalletMain->mapAddressBook.end())
+            {
+                if (!GetBoolArg("-enableaccounts", false))
+                    entry.push_back(Pair("label", item->second));
+
+                else
+                    entry.push_back(Pair("account", item->second));
```

Comment:
Why can't both label and account be shown?
Oh i see, but it could be useful.

Reply:
Because it is using the same variable `item->second`, thus the label will be same value as account.

Reply:
ya i left it as u said it would be more useful to show both and i realised i could of did a ? to show either or but it looks good.